timescaledb/quick-start/python.md

```diff
@@ -433,3 +473,8 @@ tutorials:
 [self-hosted-install]: /install/latest/self-hosted/
 [time-series-forecasting]: /timescaledb/:currentVersion:/tutorials/time-series-forecast/
 [virtual-env]: https://docs.python.org/3/library/venv.html
+[connect]: /timescaledb/:currentVersion:/quick-start/python/#connect-to-timescaledb
+[create-table]: /timescaledb/:currentVersion:/quick-start/python/#create-a-relational-table
+[create-hypertable]: /timescaledb/:currentVersion:/quick-start/python/#create-hypertable
+[insert]: /timescaledb/:currentVersion:/quick-start/python/#insert-rows-of-data
+[query]: /timescaledb/:currentVersion:/quick-start/python/#execute-a-query
```

Comment:
For anchor links on the same page, I would ditch the page path, so if you rename the page you don't have to rewrite them all. But it doesn't really matter

Reply:
So, just start with the # then? Or from the `/python/` ?

Reply:
With the `#`